robot-agent/config.js

```diff
@@ -7,12 +7,22 @@ dotenv.config({path: './.env_user'});
 global.config = {};
 const fleetConfig = {};
 
-try {
-  global.config = JSON.parse(fs.readFileSync('./config.json', {encoding: 'utf8'}));
-  console.log(`Using config:\n${JSON.stringify(global.config, true, 2)}`);
-} catch (e) {
-  console.log('No config.json file found or not valid JSON, proceeding without.');
-}
+const refreshGlobalConfig = () => {
+  try {
+    global.config = JSON.parse(fs.readFileSync('./config.json', {encoding: 'utf8'}));
+    console.log(`Using config:\n${JSON.stringify(global.config, true, 2)}`);
+  } catch (e) {
+    console.log('No config.json file found or not valid JSON, proceeding without.');
+  }
+};
+refreshGlobalConfig();
+
+fs.watchFile('./config.json', {interval: 1000}, (curr, prev) => {
+  if (curr.mtime !== prev.mtime) {
```

Comment:
This should not be necessary when using `watch`, as the eventType should already indicate whether a change occurred.